package.json

```diff
@@ -40,6 +40,10 @@
   "engines": {
     "node": ">=18.0.0"
   },
+  "peerDependencies": {
+    "@uswds/uswds": "^3.8.1",
+    "@trussworks/react-uswds": "^9.0.0"
```

Comment:
do we need react-uswds as a peer dependency?

Reply:
Good point. Thinking about it again, we should move `@trussworks/react-uswds` to regular dependencies since we're using its components directly in our library that gets bundled by Parcel. I've updated the package.json in the instance repo as well.